MAINTAINERS.md

```diff
@@ -0,0 +1,35 @@
+# Argoproj Maintainers
+
+[GOVERNANCE.md](https://github.com/argoproj/argoproj/blob/master/community/GOVERNANCE.md) describes the Argo governance.
+
+[membership.md](https://github.com/argoproj/argoproj/blob/master/community/membership.md) describes the responsibilities and requirements on the project roles. 
+
+## Maintainers
+
+| Maintainer | GitHub ID | Project Roles | Affiliation | Has Vote
+| --------------- | --------- | ----------- | ----------- | --------- | 
+| Saravanan Balasubramanian| [sarabala1979](https://github.com/sarabala1979) | Approver - Workflows | [Intuit](https://www.github.com/intuit/) | Yes |
+| Chetan Banavikalmutt | [chetan-rns](https://github.com/chetan-rns) | Approver - CD | [Red Hat](https://www.github.com/redhat/) | Yes |
```

Comment:
I think Chetan's role is Reviewer according to https://github.com/argoproj/argo-cd/blob/master/OWNERS

Reply:
Yes, I think so too.
@hblixt Please verify.

Reply:
> I think Chetan's role is Reviewer according to https://github.com/argoproj/argo-cd/blob/master/OWNERS

Correct. Fixed.